compiler/il/OMRDataTypes.hpp

```diff
@@ -293,6 +293,11 @@ namespace TR
    template <> inline int32_t getMaxUnsignedPrecision<TR::Int32>() { return 10; }
    template <> inline int32_t getMaxUnsignedPrecision<TR::Int64>() { return 20; }
 
+   float getMaxFloat();
+   float getMinFloat();
+   double getMaxDouble();
+   double getMinDouble();
```

Comment:
These look like they should be `const` APIs. Do you agree?

Reply:
I agree that the results should return values that are immutable and the function itself must not change any predefined variables. However, since these functions are not member of any class, it might not require the `const` keyword.